model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -411,15 +411,15 @@ message Parameter {
 
 message StateSpec {
   oneof spec {
-    ValueStateSpec value_spec = 1;
+    ReadModifyWriteStateSpec read_modify_write_spec = 1;
```

Comment:
@robertwb / @pabloem I am not sure whether this is the right way of doing it or I should use `reserved` instead.  I tried to use `reserved` but it complained about missing field name.

Reply:
This should be fine. The content of the proto is the same.